### DISCUSSION TITLE
[Question] Inquiry About RTX LiDAR Sensor Support in Isaac Sim

### DISCUSSION THREAD

**Discussion by junminkyung** [isaac-sim/IsaacLab#1201] [Q&A]
Hello Isaac Lab team,

I am interested in using the RTX LiDAR sensor in Isaac Sim. Is there currently any way to use the sensor within Isaac Sim, or are there any users who have created custom files to implement it? If so, would they be willing to share their solutions?

Additionally, I would like to know if there are any future plans to officially support the RTX LiDAR sensor in upcoming updates.

Thank you for your help!

**Comment by Mayankm96** [comment]
There's some work in #802 in this direction. The MR is a work-in-progress.
